docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -133,8 +133,51 @@ To request access to these resources, run
     --reason <request reason>
 ```
 
-You can search for resources of kind `node`, `kube_cluster`, `pod`, `db`, `app`, and
-`windows_desktop`. Advanced filters and queries are supported. See our
+You can search for resources of kind `node`, `kube_cluster`, `db`, `app`, and
+`windows_desktop`. Teleport also supports searching and requesting access to
+resources within Kubernetes clusters.
+
+<Details title="List of Supported Kubernetes resources">
+
+Teleport supports searching and requesting access to the following Kubernetes resources:
+
+<Admonition
+  type="tip"
+  title="Resource Access Requests to Kubernetes Namespaces"
+>
+
+Requesting access to a Kubernetes Namespace allows you to access all resources
+in that namespace including the supported resources listed above. However, it
```

Comment:
```suggestion
in that namespace, including the supported resources listed above. However, it
```

Reply:
Does "the supported resources listed above" mean "node, kube_cluster, db, app, and windows_desktop"?

Reply:
> the supported resources listed above

It was a copy typo.